doc/source/Resources/User_Guide.rst

```diff
@@ -33,7 +33,7 @@ The same result can be obtained with the following code:
 
 .. code:: python
 
-    Launch the latest installed version of AEDT in graphical mode
+    # Launch the latest installed version of AEDT in graphical mode.
```

Comment:
We should standardize this, and I'm find with either approach:

#### Use `>>>`

```py
Let's create an array of numbers.

>>> import numpy as np
>>> array = np.arange(10)
array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])

```

#### don't use `>>>`
```py

# Let's create an array of numbers.
import numpy as np
array = np.arange(10)

```

I like using `>>>` because we we can explain and show the output.

Reply:
@akaszynski I will gladly do the first--but there are a few code examples in this file, and I'm not confident on how to change them. Would you please edit these examples so I can learn from your edits? Thanks!

Reply:
Sorry @PipKat, been bogged down with other work.  I'll try to get to this today.

Reply:
Actually, let's add this in a later PR.  We can add this to the sphinx documentation in a later PR.